addons/web/static/src/views/list/list_renderer.js

```diff
@@ -1595,6 +1598,9 @@ export class ListRenderer extends Component {
 
     async toggleOptionalField(fieldName) {
         this.optionalActiveFields[fieldName] = !this.optionalActiveFields[fieldName];
+        if (this.props.optionalFieldsChanged) {
+            this.props.optionalFieldsChanged(this.optionalActiveFields);
```

Comment:
Can you please rename to `onOptionalFieldsChanged`?

Reply:
Done :+1: